arch/risc-v/src/mpfs/mpfs_serial.c

```diff
@@ -1429,6 +1429,16 @@ static void up_txint(struct uart_dev_s *dev, bool enable)
   flags = enter_critical_section();
   if (enable)
     {
+      /* If the console has been disabled after boot, don't allow enabling
```

Comment:
Fairly certain this is not upstreamable, enabling tx interrupts and console have nothing to do with eachother